### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by stuaxo**
VM wasn't playing so couldn't do this in as much detail as I would like :man_shrugging:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,26 +11,54 @@ This is not how people finder is actually deployed but provides an environment t
...
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+exec $SHELL
+
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+echo 'export PATH="$HOME/.rbenv/plugins/ruby-build/bin:$PATH"' >> ~/.bashrc
+exec $SHELL
+
+rbenv install 2.3.7
```
Comment (ztolley) [inline README.md]: 2.3.5 or 2.3.7?

**Comment by stuaxo** [comment]
Good catch.

My plan is to do an update to 2.3.7 and along with the other things SLYK has highlighted, and update The README then.

File: README.md [hunk 2]
```diff
@@ -11,26 +11,54 @@ This is not how people finder is actually deployed but provides an environment t
...
+```
+
+Clone the repo
+```git clone git@github.com:uktrade/peoplefinder.git```
+
+Install peoplefinder
+```
+cd peoplefinder
+bundle install
+sudo su postgres createuser ubuntu` (or the name of the user the application will be running as)
```
Comment (reupen) [inline README.md]: Not sure what's going on here, but it doesn't seem to run for me.

Comment (stuaxo) [reply README.md]: Let me know what error your getting, OS etc.. alternately I can come in tomorrow and help work out what's not working.

File: README.md (hunk 1 above)
Comment (stuaxo) [reply README.md]: :+1:

**Pull request merged**